proxysql-status

```diff
@@ -170,11 +175,17 @@ fi
 
 if [[ $DUMP_ALL -eq 1 || $DUMP_STATS -eq 1 ]]; then
     echo "............ DUMPING STATS DATABASE ............"
-    TABLES=$(mysql_exec -BN "SHOW TABLES FROM stats" 2> /dev/null)
+    TABLES=$(mysql_exec -BN "SHOW TABLES FROM stats" 2> /dev/null)      
     for table in $TABLES
     do
         if [[ -n $TABLE_FILTER && $table != *${TABLE_FILTER}* ]]; then
             continue
+        fi
+		# Dump _reset tables only if we specify option --with-stats-reset
+        if [[ $DUMP_STATS_RESET_TABLE -eq 0 ]]; then
+			if echo "$table" | grep -q "_reset$"; then
```

Comment:
Should use spaces instead of tabs (4 spaces per tab)

Reply:
done